src/main.js

```diff
@@ -68,7 +68,15 @@ window.WebVRConfig = Util.extend({
   // When set to true, this will cause a polyfilled VRDisplay to always be
   // appended to the list returned by navigator.getVRDisplays(), even if that
   // list includes a native VRDisplay.
-  ALWAYS_APPEND_POLYFILL_DISPLAY: false
+  ALWAYS_APPEND_POLYFILL_DISPLAY: false,
+
+  // There are scenarios where the native WebVR API exists, and instead of returning
```

Comment:
This shouldn't be true in the long term, so maybe refer to this as handling the possibility of bugs in the implementation instead of 'scenarios' which seem to imply the promise not returning can be expected in some cases.